pkg/gui/gui.go

```diff
@@ -512,6 +512,7 @@ func (gui *Gui) Run() error {
 	g.NextSearchMatchKey = gui.getKey(userConfig.Keybinding.Universal.NextMatch)
 	g.PrevSearchMatchKey = gui.getKey(userConfig.Keybinding.Universal.PrevMatch)
 
+	runewidth.DefaultCondition.EastAsianWidth = false
```

Comment:
You only need to do this once. You can use init().

Reply:
This is in a section of the code which is only run once. But I'm not sure what you mean by using init()

Reply:
```
func init() {
    runewidth.DefaultCondition.EastAsianWidth = false
}
```
This code run once and it is early.

Reply:
@mattn is that a function defined in the runewidth package? Perhaps I'm looking at an old version because I'm not seeing that. Or are you suggesting that I add such a function to my own program?

Reply:
See https://golang.org/doc/effective_go#init

Reply:
Interesting. Thanks for the tip!